detekt/baseline.xml

```diff
@@ -0,0 +1,811 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<SmellBaseline>
+  <ManuallySuppressedIssues/>
+  <CurrentIssues>
```

Comment:
by having this list in `currentIssues` will they still popup as wrong or are they being suppressed?

Reply:
They will be suppressed, but...as I'm doing a constant update of the branch, They are still constantly jumping in new issues.

As a final step, We'll have to update the baseline :D